actions/retest/main.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+// standard input env variables in action more details at
+// https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs
+var (
+	retryLimit          = os.Getenv("INPUT_MAX-RETRY")
+	requiredReviewCount = os.Getenv("INPUT_REQUIRED-APPROVE-COUNT")
+	exemptlabel         = os.Getenv("INPUT_EXEMPT-LABEL")
+	requiredlabel       = os.Getenv("INPUT_REQUIRED-LABEL")
+	githubToken         = os.Getenv("GITHUB_TOKEN")
+	owner, repo         = func() (string, string) {
+		if os.Getenv("GITHUB_REPOSITORY") != "" {
+			if len(strings.Split(os.Getenv("GITHUB_REPOSITORY"), "/")) == 2 {
+				return strings.Split(os.Getenv("GITHUB_REPOSITORY"), "/")[0], strings.Split(os.Getenv("GITHUB_REPOSITORY"), "/")[1]
+			}
+
+		}
+		return "", ""
+	}()
+)
+
+func main() {
+	if requiredlabel == "" {
+		log.Fatal("requiredlabel are not set")
```

Comment:
This should this be `INPUT_REQUIRED-LABEL` or `required-label`, currently it seems inconsistent with the other errors.